src/pages/api/outboundMatching.ts

```diff
@@ -0,0 +1,111 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { redirectTo, redirectToError, getUuidFromCookie, setCookieOnResponseObject, getDomain } from './apiUtils';
+import { putStringInS3, UserFareStages } from '../../data/s3';
+import { isCookiesUUIDMatch, isSessionValid } from './service/validator';
+import { MATCHING_DATA_BUCKET_NAME, MATCHING_COOKIE } from '../../constants';
+import { MatchingFareZones } from '../../interfaces/matchingInterface';
+import { Stop } from '../../data/auroradb';
+
+export const putMatchingDataInS3 = async (data: MatchingFareZones, uuid: string): Promise<void> => {
+    await putStringInS3(
+        MATCHING_DATA_BUCKET_NAME,
```

Comment:
The outbound data should be going in the User Data bucket, not the matching data bucket. The matching bucket is for the complete matching data which triggers the NeTEx creation